### PR TITLE
Adding more detail to the private to public

### PR DESCRIPTION
**Pull request by ColinDaglish**
I have added some step by step guidance to the private to public repo, based upon my recent experience of going through this in my own team.

### CODE REVIEW

**Comment by r-leyshon** [comment]
Hi @ColinDaglish - looking great, thanks for the input. Please could you remove any html from the repo with something like this from terminal `find . -name '*.html' -delete`.

File: private_to_public/coding-in-the-open/index.qmd
Comment (r-leyshon) [inline private_to_public/coding-in-the-open/index.qmd:1]: Was this here when you started working on the project?

**Comment by r-leyshon** [comment]
Yikes @ColinDaglish  - not sure how this is happening, but on a local render the mermaid diagram renders with a block of whitespace above it. I can't seem to see a diff that's affecting the mermaid diagram specifically. Do you get a render like the below screengrab on your end?
image

**Comment by ColinDaglish** [comment]
> Yikes @ColinDaglish - not sure how this is happening, but on a local render the mermaid diagram renders with a block of whitespace above it. I can't seem to see a diff that's affecting the mermaid diagram specifically. Do you get a render like the below screengrab on your end? image

@r-leyshon no i don't have that issue, but my mermaid diagram doesn't seem to render anymore. 

image

**Comment by r-leyshon** [comment]
Resolves #2

**Pull request merged**